packages/api/src/beacon/routes/validator.ts

```diff
@@ -785,3 +791,7 @@ export function getReturnTypes(): ReturnTypes<Api> {
     getLiveness: jsonType("snake"),
   };
 }
+
+function parseBuilderBoostFactor(builderBoostFactorInput?: string | number | bigint): bigint | undefined {
+  return builderBoostFactorInput !== undefined ? BigInt(builderBoostFactorInput) : undefined;
```

Comment:
Nit: Do you think we need to guard and throw explicit error when someone passes in a negative number here? Not that it matters to the result of block value comparison

Reply:
if there are any checks we need to add, we should add in the basic serializers for the types they are serializing/deserializing, we can look at it as a separate PR